toolsWebsite/tools/templates/tools/checkout.html

```diff
@@ -0,0 +1,101 @@
+    <script src="https://use.fontawesome.com/fbcbee1ea9.js"></script>
+    <title>Projects</title>
+  </head>
+
+  <body class="d-flex flex-column">
+    <!-- Navbar Start -->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+      <div class="container">
+        <a class="navbar-brand" href="/tools/">Tool Co-op</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ml-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="/tools/">All Tools <span class="sr-only">(current)</span></a>
+            </li>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Tools
+              </a>
+              <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+                <a class="dropdown-item" href="/tools/available">Available</a>
+                <a class="dropdown-item" href="/tools/checked_out">Checked Out</a>
+                <a class="dropdown-item" href="/tools/overdue">Overdue</a>
+              </div>
+            </li>
+            <li class="nav-item active">
+              <a class="nav-link" href="/tools/project">Projects</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">History</a>
+            </li>
+            <li class="nav-item">
+              <ul class="navbar-nav ml-auto ml-md-0">
+                <li class="nav-item dropdown no-arrow">
+                  <a class="nav-link dropdown-toggle" href="#" id="cartDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    <i class="fa fa-cart-arrow-down"></i>
+                  </a>
+                  <div class="dropdown-menu dropdown-menu-right" aria-labelledby="cartDropdown">
+                    <div id="cartDropdownItems"></div>
+                    <a href="/checkout">
+                      <button class="btn btn-outline-info my-2 my-sm-0 right shoppingDropdownButton">
+                        Check Out
+                      </button>
+                    </a>
+                  </div>
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <!-- Navbar End -->
+      <div style="text-align: center;">
+          <br>
+          <br>
+          <h1>Checkout Successful!</h1>
+          <br>
+          <a class="nav-link" href="/tools/index">Return to Homepage<span class="sr-only">(current)</span></a>
+      </div>
```

Comment:
should we addd to this the list of items that were checked out and the total amount the transaction would be? I personally feel like this should be a checkout review page before the checkout successful page, just so that people can review what items they are about to checkout and then hit a confirm checkout button to take them to a checkout successful page. But we can talk about this.

Reply:
That's fine with me.  I can add that to the requirements.

Reply:
Let's talk about this in our final sprint meeting.  I'd like everyone's input.  I do think it is a good idea since all websites seem to have this.